bigtable/google/cloud/bigtable/table.py

```diff
@@ -312,18 +317,25 @@ def mutate_rows(self, rows):
                   corresponding to success or failure of each row mutation
                   sent. These will be in the same order as the `rows`.
         """
-        mutate_rows_request = _mutate_rows_request(self.name, rows)
-        client = self._instance._client
-        responses = client._data_stub.MutateRows(mutate_rows_request)
-
-        responses_statuses = [
-            None for _ in six.moves.xrange(len(mutate_rows_request.entries))]
-        for response in responses:
-            for entry in response.entries:
-                responses_statuses[entry.index] = entry.status
-                if entry.status.code == 0:
-                    rows[entry.index].clear()
-        return responses_statuses
+        delay_millis = 1000
+        delay_mult = 1.5
+        max_delay_millis = 15 * 1000
+        total_timeout_millis = 5 * 60 * 1000
```

Comment:
should these be configureable?

Reply:
I intend them to be configurable, but wanted to check if we want it to be user-configurable or pre-configured at API level. 
- For the former, we may end up taking in an optional backoff/retry option at `table.mutate_rows`. 
- For the latter, a module tuple would be good if we see other (retryable) requests to share the same values for retry.